Lib/test/audit-tests.py

```diff
@@ -567,6 +567,15 @@ def hook(event, args):
     _winapi.CreateNamedPipe(pipe_name, _winapi.PIPE_ACCESS_DUPLEX, 8, 2, 0, 0, 0, 0)
 
 
+def test_assert_unicode():
+    import sys
+    sys.addaudithook(lambda *args: None)
+    try:
+        sys.audit(9)
+    except:
+        pass
```

Comment:
```suggestion
    import sys
    sys.addaudithook(lambda *args: None)
    try:
        sys.audit(9)
    except TypeError:
        pass
```

1) Let's not use bare excepts 2) let's propagate exceptions that we did not expect